test/author_test.rb

```diff
@@ -0,0 +1,29 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/book'
+require './lib/author'
+
+class AuthorTest < MiniTest::Test
+
+  def setup
+    @charlotte_bronte = Author.new({first_name: "Charlotte", last_name: "Bronte"})
+    @book = Book.new({author_first_name: "Charlotte", author_last_name: "Bronte", title: "Jane Eyre", publication_date: "October 16, 1847"})
+    @book2 = Book.new({author_first_name: "Charlotte", author_last_name: "Bronte", title: "Villette", publication_date: "1853"})
+  end
+
+  def test_it_exists
+    assert_instance_of Author, @charlotte_bronte
+  end
+
+  def test_it_has_attributes
+    assert_equal "Charlotte Bronte", @charlotte_bronte.name
+    assert_equal [], @charlotte_bronte.books
+  end
+
+  def test_write
```

Comment:
Here, I would test that the book is created as I expect, verifying that the title, author and year are all being written correctly.  This would have helped you identify that you're `write` method wasn't really working.